scripts/collector/mysql/collect-data.sh

```diff
@@ -214,20 +214,14 @@ if ! [ -x "$(command -v ${SQLCMD})" ]; then
 fi
 
 export DMA_SOURCE_ID=$(${SQLCMD} --user=$user --password=$pass -h $host -P $port --force --silent --skip-column-names $db < sql/init.sql | tr -d '\r')
-export DMA_PASSWORD_COL=$(${SQLCMD} --user=$user --password=$pass -h $host -P $port --force --silent --skip-column-names $db < sql/password_column.sql | tr -d '\r')
 
-
-if [ -f sql/${V_FILE_TAG}_mysqlcollector.sql ]; 
-then
-rm sql/${V_FILE_TAG}_mysqlcollector.sql
-fi
-
-for f in $(ls -1 sql/*.sql | grep -v -e _mysqlcollector.sql -e init.sql -e password_column.sql -e usersno${DMA_PASSWORD_COL})
+for f in $(ls -1 sql/*.sql | grep -v -e init.sql)
 do
   fname=$(echo ${f} | cut -d '/' -f 2 | cut -d '.' -f 1)
-    ${SQLCMD} --user=$user --password=$pass -h $host -P $port --force --table  ${db} >output/opdb__${fname}__${V_TAG} <<EOF
+    ${SQLCMD} --user=$user --password=$pass -h $host -P $port --force --table  ${db} >output/opdb__mysql_${fname}__${V_TAG} <<EOF
 SET @DMA_SOURCE_ID='${DMA_SOURCE_ID}' ; 
 SET @DMA_MANUAL_ID='${V_MANUAL_ID}' ;
+SET @PKEY='${V_FILE_TAG}';
 source ${f}
 exit
 EOF
```

Comment:
every CSV at the end has another DMA_SOURCE_ID|MANUAL_ID
I think its in line 230

Reply:
and its with `'` instead of `"` but i rather it not be there

Reply:
@liozme, I'm not sure if this is still an issue, but I'll give it a test this afternoon and update accordingly.